### PR TITLE
68 saving my favourite services

### PR DESCRIPTION
**Pull request by Nikomus**
https://trello.com/c/XvPnxWQ9/68-saving-my-favourite-services-vv-eee

- Unstyled nav link to be removed in the ticket for navigation
- We could keep track of the previous path somewhere so that we can give this to a link. For not I'm just using router.back() so that we return to the correct list

### CODE REVIEW

File: pages/saved.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          costQualifier: service.costQualifier,
+        }),
+
+        ...(service.format && {
+          format: service.format,
+        }),
+      }
+      return servicePreview
+    }
+  )
```
Comment (jthistle) [inline pages/saved.tsx:61]: Isn't this just all copy-pasted over from the `index.tsx` page? Is there any way we can reduce duplication here?

File: src/Components/EmptyList.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            viewBox="0 0 24 24"
+            stroke="currentColor"
+          >
+            <path
+              strokeLinecap="round"
+              strokeLinejoin="round"
+              strokeWidth="1"
+              d="M11.049 2.927c.3-.921 1.603-.921 1.902 0l1.519 4.674a1 1 0 00.95.69h4.915c.969 0 1.371 1.24.588 1.81l-3.976 2.888a1 1 0 00-.363 1.118l1.518 4.674c.3.922-.755 1.688-1.538 1.118l-3.976-2.888a1 1 0 00-1.176 0l-3.976 2.888c-.783.57-1.838-.197-1.538-1.118l1.518-4.674a1 1 0 00-.363-1.118l-3.976-2.888c-.784-.57-.38-1.81.588-1.81h4.914a1 1 0 00.951-.69l1.519-4.674z"
+            />
+          </svg>{' '}
```
Comment (jthistle) [inline src/Components/EmptyList.tsx:47]: can we make this star svg a reusable component, maybe?

File: src/Components/SaveButton.tsx [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+        viewBox="0 0 24 24"
+        stroke="currentColor"
+      >
+        <path
+          strokeLinecap="round"
+          strokeLinejoin="round"
+          strokeWidth="1"
+          d="M11.049 2.927c.3-.921 1.603-.921 1.902 0l1.519 4.674a1 1 0 00.95.69h4.915c.969 0 1.371 1.24.588 1.81l-3.976 2.888a1 1 0 00-.363 1.118l1.518 4.674c.3.922-.755 1.688-1.538 1.118l-3.976-2.888a1 1 0 00-1.176 0l-3.976 2.888c-.783.57-1.838-.197-1.538-1.118l1.518-4.674a1 1 0 00-.363-1.118l-3.976-2.888c-.784-.57-.38-1.81.588-1.81h4.914a1 1 0 00.951-.69l1.519-4.674z"
+        />
+      </svg>
```
Comment (jthistle) [inline src/Components/SaveButton.tsx:65]: ditto

**Review by jthistle — COMMENTED** [review]
Love the way this looks! :star2:

Just a couple of comments.

**Comment by marxian** [comment]
#### 📝Changed routes:


| [**/**](https://my-best-life-9wvbnquc0-neontribe.vercel.app/) | [**/saved**](https://my-best-life-9wvbnquc0-neontribe.vercel.app/saved) |
|:-:|:-:|
|<a href="https://my-best-life-9wvbnquc0-neontribe.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/VdnvMwRf51931aOGkBy_KxppJ1w_.png">(view full size)</a> |<a href="https://my-best-life-9wvbnquc0-neontribe.vercel.app/saved">Screenshot of /saved</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/XA0c1hNmGnHbpuOJek61lLL_YMI_.png">(view full size)</a> |



| [**/service/\[id\]**](https://my-best-life-9wvbnquc0-neontribe.vercel.app/service/[id]) |
|:-:|
|<a href="https://my-best-life-9wvbnquc0-neontribe.vercel.app/service/[id]">Screenshot of /service/[id]</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/RRNFbPye4cASAkufgUJv2ye5xUQ_.png">(view full size)</a> |







Commit 83de6cd7b4e6efc61eb2bc861bc9fc0e23a5d785 (https://my-best-life-9wvbnquc0-neontribe.vercel.app).

File: pages/saved.tsx (hunk 1 above)
Comment (Nikomus) [reply pages/saved.tsx:61]: It is.. I think I'm happy to wait until we know things won't be different to optimise here.

File: src/Components/EmptyList.tsx (hunk 2 above)
Comment (Nikomus) [reply src/Components/EmptyList.tsx:47]: Similarly, there's discussion on changing the symbol so save/rate aren't confusing, if rating appear in more places. I see you carded this too :)

**Review by edpe — APPROVED** [review]
Looks great :star:

**Pull request merged**